pkgs/development/python-modules/sphinx-pallets-theme/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchFromGitHub, sphinx, packaging }:
+
+buildPythonPackage rec {
+  pname = "Pallets-Sphinx-Themes";
+  version = "2.0.2";
+
+  src = fetchFromGitHub {
+    owner = "pallets";
+    repo = "pallets-sphinx-themes";
+    rev = version;
+    sha256 = "0nvznv6abmkkda2fahydd4rykd94rmz74hx5aypv6j22zvf5pj8b";
+  };
```

Comment:
can you use `fetchPypi`?

Reply:
I think it is strictly worse. Git repository is the source, pypi archive is derivative work. There was discussion about it
in https://github.com/NixOS/nixpkgs/pull/150251 and https://github.com/NixOS/nixpkgs/pull/142236.